### DISCUSSION TITLE
Pre-filled form view [May 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2211] [Discord Threads]
> 🙋‍♂️ **Vlas**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### not possible at the moment  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/978277705760006225)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Reply by airhuman** [reply]
> > 💁 **o1lab**
> 
> #### not possible at the moment
> [Autogenerated from discord](https://discord.com/channels/661905455894888490/978277705760006225)
> 
> #### Join [NocoDB's](http://www.nocodb.com/) community
> [Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

Is still isn't possible?
